modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/problem/ProblemMapper.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.problem;
+
+/**
+ * @author Alejandro Tardín
+ */
+public interface ProblemMapper<T extends Throwable> {
```

Comment:
Why are you including in the ProblemProvider the @ProviderType annotation and not here?

Reply:
The @ProviderType goes into the framework interface to allow us to make breaking changes. This interface is meant to be implemented by external modules so it should follow the standard semver rules.

Reply:
You mean external customers?